app/controllers/taxons_controller.rb

```diff
@@ -0,0 +1,37 @@
+class TaxonsController < ApplicationController
+  before_action :redirect_to_www, unless: :new_navigaton_enabled?
+
+  def show
+    render :show, locals: {
```

Comment:
I totally missed that it's mostly from `taxon` that are being passed around, so calling those directly from the view would be nice.

I'm still troubled by the introduction of a new pattern here. In content-tagger the same happened, but we haven't followed through on moving everything to it, causing inconsistency.  Any thoughts around that?